test/assert.js

```diff
@@ -530,6 +530,48 @@ describe('assert', function () {
     }, "blah: expected {} to be a function");
   });
 
+  it('isCallable', function() {
```

Comment:
Very nitpicky from me, and I am not the maintainer of this lib.

Right now these tests are placed right in between `isFunction` and `isNotFunction`. I would prefer the more generic `isCallable` to be placed before the `isFunction` and the other ones after `isNotFunction`. This would make it easier for non-regular contributors to find specifics part of the code. Not sure if you guys agree.